apps/next-evans/lib/get-custom-code.ts

```diff
@@ -0,0 +1,36 @@
+import { LANG } from './env';
+
+export default function getCustomCode(
+  friendSlug: string,
+  documents: string[],
+): Promise<Record<string, { css?: string; html?: string }>> {
+  return Promise.all(
+    documents.map((documentSlug) =>
+      Promise.all([
+        getCode(friendSlug, documentSlug, `css`),
+        getCode(friendSlug, documentSlug, `html`),
+      ]).then(([css, html]) => ({ css, html, slug: documentSlug })),
+    ),
+  ).then((codes) =>
+    codes.reduce((acc: Record<string, { css?: string; html?: string }>, code) => {
+      acc[code.slug] = { css: code.css, html: code.html };
+      return acc;
+    }, {}),
+  );
+}
+
+async function getCode(
+  friend: string,
+  document: string,
```

Comment:
same here, let's change to `friendSlug` and `documentSlug` for greater clarity at the call site